js/components/interface/query/query.less

```diff
@@ -501,3 +501,26 @@
 #querybuilder .griddle-container{
     background-color: fade(@background_color_widget, 70%);
 }
+
+#closeQuery {
+    float: right;
```

Comment:
Those css can be improved. If the need is to move the control on the right and center it someway the flexbox is magic. Can justify and vertical-align very easily. I don't think it's wortth spending time with this now anyway, I suppose there will be big changes on modals once we move from jquery and bootstrap

Reply:
To be fair using vertical-align imply to re-do the stilling for the whole component since I tried but for how the styling is organised vertical-align does not work.
Following the considerations done that were mine as well I will mark this resolved.